src/main/java/org/broad/igv/feature/genome/Genome.java

```diff
@@ -641,7 +641,7 @@ public String getDisplayName() {
      * Return the reference base at the given position.
      * */
     public byte getReference(String chr, int pos) {
-        return sequence.getBase(chr, pos);
+        return sequence == null ? 0 : sequence.getBase(chr, pos);
```

Comment:
Ah, fair.  It's definitely a path that's never been called because it was guaranteed to explode. I'm not sure if returning an invalid base is better.  Maybe we should throw a deliberate exception.